psycop/common/model_training_v2/config/config_utils.py

```diff
@@ -14,15 +14,21 @@
 populate_baseline_registry()
 
 
-def load_baseline_config(config_path: Path) -> BaselineSchema:
-    """Loads the baseline config from disk and resolves it."""
+def resolve_and_fill_config(config_path: Path) -> dict[str, Any]:
```

Comment:
add flag overwrite_config.

Reply:
Could imagine you might want to disable the behaviour (it is nice to know that the config is being overwritten).